### PR TITLE
Jfb 4

### PR DESCRIPTION
**Pull request by ya-kool**

### CODE REVIEW

File: src/main/java/by/yahor_kulesh/model/Printable.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package by.yahor_kulesh.model;
+
+public interface Printable {
+    default String print(){
+        return this.toString();
```
Comment (AtticSiol) [inline src/main/java/by/yahor_kulesh/model/Printable.java]: if the method called as "print", then it has to print smth -> System,out.println(this.toString());
if the method returns value for printing, it has to be call for example "getPrintValue".

File: src/main/java/by/yahor_kulesh/model/users/Client.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import java.util.HashSet;
+import java.util.Set;
+
+public class Client extends User {
+    private final Set<Ticket> tickets = new HashSet<>();
+
+    public void getTicket(Ticket ticket) {
+        System.out.println("Client got ticket:" + ticket.getId() + "\n");
+        tickets.add(ticket);
+        this.print();
```
Comment (AtticSiol) [inline src/main/java/by/yahor_kulesh/model/users/Client.java]: do not mix logic. single method single responsability. if you need to print smth - call it as next command in service/app

File: src/main/java/by/yahor_kulesh/App.java [hunk 3]
```diff
@@ -1,46 +1,37 @@
...
+import by.yahor_kulesh.model.tickets.*;
+import by.yahor_kulesh.model.users.*;
 import by.yahor_kulesh.validators.InputValidator;
 
+import java.math.BigDecimal;
+import java.time.ZoneId;
 
-public class App {
+
+public class App extends Data {
```
Comment (AtticSiol) [inline src/main/java/by/yahor_kulesh/App.java:9]: why did you remove all logic from TicketService and move it into new class App?

Comment (ya-kool) [reply src/main/java/by/yahor_kulesh/App.java:9]: In my opinion TicketService is used to store logic to work with Tickets only, not common methods and main. Main method needs to be stored in separate class which must be starting point of application. In that case i moved all this logic to App.
Now i moved it back to TicketService class and call it from App

Comment (ya-kool) [reply src/main/java/by/yahor_kulesh/App.java:9]: I resolved others problems too

Comment (AtticSiol) [reply src/main/java/by/yahor_kulesh/App.java:9]: Yes, agree, but in this case, TicketService, if it presents into the app, has to do smth, it can't be empty. In your case you just moved all from TicketService to App, after that you moved all from App to TicketService. Looks like one of these classes is redundant here.

File: src/main/java/by/yahor_kulesh/App.java [hunk 4]
```diff
@@ -1,46 +1,15 @@
...
-                }
-                case 0: {
-                    return;
-                }
-                default: {
-                    System.out.println("Try again!");
-                }
-            }
-        }
+	    TicketService.applicationMethod();
```
Comment (AtticSiol) [inline src/main/java/by/yahor_kulesh/App.java]: "applicationMethod" - is too general name. It is not clear what exactly will be done when we call it. If it is void and returns nothing, it has to be named in verb form.

**Review by AtticSiol — APPROVED** [review]
